docs/deploy-configure.md

```diff
@@ -5,6 +5,7 @@ The [`/deploy/crds`](../deploy/crds) directory contains example MongoDB resource
 ## Table of Contents
 
 - [Deploy a Replica Set](#deploy-a-replica-set)
+- [Scale a Replica Set](#-scale-a-replica-set)
```

Comment:
This link isn't working. Is there an extra - after the # ?

Reply:
good catch. updated